### ISSUE TITLE
Has Google Collab banned the use of Stable Diffusion webui? For Pro and Pro+ users too ? 

### ISSUE THREAD

**Issue by HelpANewbie** [TheLastBen/fast-stable-diffusion#2149]

**Comment by TheLastBen** [comment]
for free colab, it's definitely not allowed, for pro, I'm not sure, but it seems they are trying to limit it too, I recommend using the alternatives

**Comment by Ptentacle1** [comment]
I'm beginning to suspect this is the case as well, because I keep getting google drive errors which is the same error free colab users are getting when they try to run SD, and I'm a Colab Pro user. I won't be one for long, however, if this is truly the case.

**Comment by Yardanni** [comment]
I feel the same way, I think they're trying to limit it due to the high resource consumption it entails. Do you know of any alternatives to Google Colab?

**Comment by Taikakim** [comment]
I've been getting a lot of "A Google drive error has occured" messages, but it's definitely not that. Their rep explicitly wrote on Reddit that Pro/Pro+ are good to go, and they're just trying to direct some traffic off the free tier and specifically the few most popular notebooks.

**Comment by cosmicoxytocin** [comment]
> for free colab, it's definitely not allowed, for pro, I'm not sure, but it seems they are trying to limit it too, I recommend using the alternatives

Appears they're slowly working towards blocking for Pro and Pro+ users as well. Google now gives a warning when trying to execute the install cell. Doesn't seem to disconnect in the way free tier users experience, **yet**.

Probably a smart move to highly recommend the alternatives.

**Comment by cosmicoxytocin** [comment]
> 

Could you provide a link to the reps response? I believe, I would just like to see it for myself.
👍

**Comment by Ptentacle1** [comment]
Well I've tried to use Runpod and my card keeps getting declined when I try to buy credits. Is Runpod based outside the US? If that's the case, my bank is likely flagging it as unsafe and not letting the transaction go through, so right now that's not an option for me.

**Comment by Ptentacle1** [comment]
@kende I tried this (though I swear I did this a dozen times already before), and it's working for me now. We'll see how long it lasts. I'm still getting google drive errors occasionally.

**Comment by kende** [comment]
> @kende I tried this (though I swear I did this a dozen times already before), and it's working for me now. We'll see how long it lasts. I'm still getting google drive errors occasionally.

I keep getting google drive errors often for the past week or so, but not noticing any direct impact from them most of the time

**Comment by Rauferp** [comment]
For those who saw the Google I/O 2023 event, they could find out that Colab and the Colab Pro want to allocate them more than anything for exclusive Google app and artificial intelligence.
SD is direct competition for Google apps and its new business partner, Adobe Firefly.
you can search on youtube "Google Keynote (Google I/O ‘23)"

**Comment by bayoumedic** [comment]
Yesterday I tried out runpod and love it.  The cheapest server (under community cloud tab) happens to be the same price per hour as colab pro and you will be generating images twice as fast. 

 I haven't figured out how to mount Google drive yet but to load your model just use the Google drive shared link straight to the model.  Runpod has a storage option for cheaply storing your models so you won't have to upload it from Google drive every time. 

Saying goodbye to colab

**Comment by Taikakim** [comment]
> > 
> 
> Could you provide a link to the reps response/statement? I believe, I would just like to see it for myself. 👍

No, I don't have it anywhere, but maybe you can search for it, it was Chris Perry who is a really nice person to deal with, but not sure what his nickname was on Reddit.
